packages/frontendmu-nuxt/pages/speaker/[id].vue

```diff
@@ -19,18 +37,21 @@ function getSpeaker(id: string | number) {
     return id === session_speaker_id
   })
 
+  const profile = speakersProfileResponse.find(profile => profile.id === speaker.id)
```

Comment:
@christopher-adolphe we should do the speaker matching using the Github Username instead of the id. This is the internal directus id and  probably won't be known by speakers. the github username is unique, and we add it for speakers in the database. So it should be sufficient to make the relationship

Reply:
@MrSunshyne correction done